openshift/job-build-analyse-template.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: build-analyse
+  annotations:
+    description: "Thoth: Build Log Analysis Analyse"
+    openshift.io/display-name: "Thoth: Build Log Analysis Analyse"
+    version: 0.1.0
+    tags: thoth,ai-stacks,build-analyser
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run recommendation logic of Thoth to OpenShift.
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+  labels:
+    app: thoth
+    template: build-analyse
+    component: build-analyse
+
+parameters:
+  - name: IMAGE_STREAM_REGISTRY
+    description: Registry where the ImageStream is present.
+    displayName: ImageStream Registry
+    required: true
+    value: "docker-registry.default.svc:5000"
+
+  - name: IMAGE_STREAM_TAG
+    description: Tag of the ImageStream to be used.
+    displayName: ImageStream Tag
+    required: true
+    value: "latest"
+
+  - name: THOTH_BUILD_ANALYSER_LOG_PATH
+    description: Path for the build log read/write.
+    displayName: Log Path
+    required: false
+    value: "/opt/app-root/src/build.log"
+
+  - name: THOTH_BUILD_ANALYSER_OUTPUT_FORMAT
+    description: Required Output format.
+    displayName: ImageStream Tag
+    required: false
+    value: "json"
+
+  - name: IMAGE_STREAM_NAMESPACE
+    description: Project/Namespace to be used for deployment.
+    displayName: ImageStream Project
+    required: true
+
+  - name: THOTH_BUILD_ANALYSER_JOB_ID
+    description: A unique dentifier of build-analyser job.
+    displayName: Build Analyser id
+    required: true
+
+  - name: THOTH_BUILD_LOG_DOC_ID
+    description: Document id of Build log present in ceph.
+    displayName: Build Log Document ID
+    required: true
+
+  - name: THOTH_REPORT_OUTPUT
+    description: Remote where results should be send to.
+    displayName: ImageStream Tag
+    required: false
```

Comment:
Is this always required in the deployment (result-api?)? Here and bellow as well.

Reply:
yes, this should have been required, I forgot to change it after testing.